src/libraries/System.Xml.ReaderWriter/ref/System.Xml.ReaderWriter.cs

```diff
@@ -271,6 +273,29 @@ public partial class XmlConvert
         public static string? VerifyTOKEN(string? token) { throw null; }
         public static string VerifyWhitespace(string content) { throw null; }
         public static string VerifyXmlChars(string content) { throw null; }
+        public static bool TryFormat(bool value, Span<char> destination, out int charsWritten) { throw null; }
```

Comment:
Should I open review item here?

Reply:
All new public APIs go through [API review process](https://github.com/dotnet/runtime/blob/main/docs/project/api-review-process.md) AFAIK.

Reply:
@krwq Do you think if this will be useful for people ? 
If not I can make them internal.
Otherwise I can open Api Proposal

Reply:
honestly XML APIs have existed since almost beginning of .NET which makes it 10+ years, I'd start with "no" unless i.e. issue gets large number of upvotes and is simple to implement

Reply:
so internal for now IMO

Reply:
Fair enough. I will make them internal.